vql/linux/tcpsnoop/tcpsnoop.go

```diff
@@ -0,0 +1,115 @@
+//go:build linux
```

Comment:
This line needs to be removed or adjusted to also have ,linuxbpf otherwise  tcpsnoop.go is always being built.

Reply:
Ack